zaza/openstack/charm_tests/hacluster/tests.py

```diff
@@ -69,3 +74,98 @@ def test_920_put_in_maintenance(self):
 
         self._toggle_maintenance_and_wait('true')
         self._toggle_maintenance_and_wait('false')
+
+
+class HaclusterScalebackTest(HaclusterBaseTest):
+    """hacluster scaleback tests."""
+
+    _PRINCIPLE_APP_NAME = 'keystone'
+    _HACLUSTER_APP_NAME = 'hacluster'
```

Comment:
This should really use [tests_options](https://github.com/openstack-charmers/zaza-openstack-tests/blob/master/zaza/openstack/charm_tests/policyd/tests.py#L82) to configure the app name